exovetter/main.py

```diff
@@ -0,0 +1,57 @@
+
+#A sketch of an architecture. For discussion
+
+#Each vetting test is wrapped in a class that follows this
+#structure.
+class Vetter():
+    def __init__(self, **kwargs):
+        """kwargs stores the configuration parameters common to all TCEs
+        For example, for the Odd-even test, it might specify the signficance
+        of the depth difference that causes a TCE to fail
+        """
+        pass
+
+    def __call__(self, tce, lightcurve):
```

Comment:
Do we really need `__call__` to wrap around `run`? Maybe we should just get rid of this. Too many ways to do the same thing can be confusing.